test/apps/iodemo/io_demo.cc

```diff
@@ -990,7 +989,8 @@ class DemoClient : public P2pDemoCommon {
             }
 
             if (_active_servers.empty()) {
-                LOG << "All servers are disconnected";
+                LOG << "All remote servers are down, reconnecting in "
```

Comment:
Actually if we can't connect to any server, most likely this is local network issue, but servers can be up and running.